lib/ecto/query.ex

```diff
@@ -476,16 +476,28 @@ defmodule Ecto.Query do
   than one where expression, they are combined with an `and` operator. All
   where expressions have to evaluate to a boolean value.
 
+  `where` also accepts a keyword list where the field given as key is going to
+  be compared with the given value. The fields will always refer to the source
+  given in `from`.
+
   ## Keywords example
 
       from(c in City, where: c.state == "Sweden")
+      from(c in City, where: [state: "Sweden"])
+
+  It is also possible to interpolate the whole keyword list, allowing you to
+  dynamically filter the source:
+
+      filters = [state: "Sweden"]
+      from(c in City, where: ^filters)
```

Comment:
Documentation added. It seems the docs for `order_by` already had a note about interpolating a whole keyword list.